integrations/LinkedInInsightTag/browser.js

```diff
@@ -1,12 +1,14 @@
 /* eslint-disable class-methods-use-this */
 import logger from "../../utils/logUtil";
 import ScriptLoader from "../ScriptLoader";
+import { getHashFromArrayWithDuplicate } from "../utils/commonUtils";
 import { NAME } from "./constants";
 
 class LinkedInInsightTag {
   constructor(config) {
     this.name = NAME;
     this.partnerId = config.partnerId;
+    this.eventConversionId = config.eventConversionId;
```

Comment:
Update this variable name to `eventToConversionIdMap`

Reply:
updated